src/components/Home.tsx

```diff
@@ -75,7 +78,22 @@ export default function Home({ organizations, selectedOrganization, setSelectedO
     const populateUser = async () => {
       const response = await getUser();
       if (response.requestSucceeded && !cancel) {
-        setUser(response.user ?? undefined);
+        if (response.user) {
```

Comment:
I don't really know where a good place for this might be.
One way to to do this inside the UserProvider, have a useEffect that depends on user..
and check there.